CHANGELOG.md

```diff
@@ -2,6 +2,105 @@
 
 This contains only the most important and/or user-facing changes; for a full changelog, see the commit history.
 
+## [2.0.0](https://github.com/ably/ably-js/tree/2.0.0) (date TBD)
+
+The 2.0.0 release introduces a number of new features and QoL improvements, including a new way to remove bloat and reduce the bundle size of your ably-js client, first-class support for Promises, a more idiomatic approach to using ably-js' React Hooks, enhancements to TypeScript typings, and more.
+
+Below is a list of all new features, APIs, deprecations, and breaking changes.
+Read [ably-js v2 migration guide](link TBD) for more information.
+
+### Modular variant of the library
+
+An ESM variant of the library is now available for browsers (but not for Node.js) at `ably/modules`. This modular variant of the library supports tree shaking, allowing for a reduction in the Ably bundle size within your application. It can also be used by Web Workers.
```

Comment:
Now it's `ably/modular`

Reply:
changed in https://github.com/ably/ably-js/pull/1656/commits/a57271c43e82d4be53e081dcaa0e33d054026e49